scripts/patches-build.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
+
+async function getPatches() {
+  console.log('Getting patches...');
+  return new Promise((resolve, reject) => {
+    fs.readdir('./dist/patches', (err, files) => {
+      if (err) reject(err);
+      console.log('Got patches.');
+      resolve(files);
+    });
+  });
+}
+
+async function updatePatchFile(file) {
+  console.log(`Updating ${file}...`);
+  return new Promise((resolve, reject) => {
+    fs.readFile(`./dist/patches/${file}`, 'utf8', (err, data) => {
+      if (err) reject(err);
+      const newData = data.replaceAll('node_modules', '..');
+      fs.writeFile(`./dist/patches/${file}`, newData, 'utf8', (err) => {
+        if (err) reject(err);
+        console.log(`Updated ${file}`);
+        resolve();
+      });
+    });
+  });
+}
```

Comment:
@Harasz It looks we're basically writing our own `patch-package` here? Is this a fair assessment?

Reply:
Ah okay, gotcha. Maybe I'm just bad at reading code but I think that maybe a comment can't hurt
```suggestion
/**
 * Updates the patchFile so that patch-package can notice them from node_modules directory
 */
async function updatePatchFile(file) {
  console.log(`Updating ${file}...`);
  return new Promise((resolve, reject) => {
    fs.readFile(`./dist/patches/${file}`, 'utf8', (err, data) => {
      if (err) reject(err);
      const newData = data.replaceAll('node_modules', '..');
      fs.writeFile(`./dist/patches/${file}`, newData, 'utf8', (err) => {
        if (err) reject(err);
        console.log(`Updated ${file}`);
        resolve();
      });
    });
  });
}
```

Reply:
maybe I am bad at writing or documenting a code :smile: